orange/src/test/java/com/hartwig/hmftools/orange/algo/cuppa/CuppaDataFactoryTest.java

```diff
@@ -125,6 +126,24 @@ public void canGetCorrectSvFeatureValueFromFileWithoutRna() throws Exception
         assertEquals(expectedFeatureValue, featureValue);
     }
 
+    @Test
+    public void canAssignCuppaModeFromFileWithoutRna() throws Exception
+    {
+        CuppaPredictions cuppaPredictions = CuppaPredictions.fromTsv(CUPPA_VIS_DATA_WITHOUT_RNA_TSV);
+        CuppaMode mode = CuppaDataFactory.getCuppaMode(cuppaPredictions);
+        CuppaMode expectedMode = CuppaMode.WGS;
+        assertEquals(expectedMode, mode);
+    }
+
+    @Test
+    public void canAssignCuppaModeFromFileWithRna() throws Exception
+    {
+        CuppaPredictions cuppaPredictions = CuppaPredictions.fromTsv(CUPPA_VIS_DATA_WITH_RNA_TSV);
+        CuppaMode mode = CuppaDataFactory.getCuppaMode(cuppaPredictions);
+        CuppaMode expectedMode = CuppaMode.WGTS;
+        assertEquals(expectedMode, mode);
+    }
+
```

Comment:
let's add a test for the UNKNOWN (or exception, see previous comment) case as well.